opm-test.go

```diff
@@ -0,0 +1,191 @@
+package main
+
+import (
+	"context"
+	"database/sql"
+	"io/ioutil"
+	"log"
+	"math/rand"
+	"os"
+	"os/exec"
+	"time"
+
+	"github.com/operator-framework/operator-registry/pkg/lib/bundle"
+	"github.com/operator-framework/operator-registry/pkg/lib/indexer"
+	"github.com/operator-framework/operator-registry/pkg/sqlite"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	DOCKER_USERNAME = os.Getenv("DOCKER_USERNAME")
+	DOCKER_PASSWORD = os.Getenv("DOCKER_PASSWORD")
+
+	packageName    = "prometheus"
+	channels       = "preview"
+	defaultChannel = "preview"
+
+	bundlePath1 = "manifests/prometheus/0.14.0"
+	bundlePath2 = "manifests/prometheus/0.15.0"
+	bundlePath3 = "manifests/prometheus/0.22.2"
+
+	bundleTag1 = StringWithCharset(6)
```

Comment:
Are we generating random tags here just to account for concurrently running tests? Is this sufficiently random for that case?

Reply:
I mean the individual probability of collision is like 1 in 36^6. Even if there are 5000 PRs up (each computing 3 random strings) the probability of collision is about 6%.